src/EntryPoints/Specials/SpecialManageApprovers.php

```diff
@@ -33,8 +33,6 @@ public function execute( $subPage ): void {
 		$request = $this->getRequest();
 		if ( $request->wasPosted() ) {
 			$this->handlePostRequest( $request );
-			$this->getOutput()->redirect( $this->getPageTitle()->getLocalURL() );
-			return;
```

Comment:
Why this change?

Reply:
This is used to redirect the user back to the special page. We needed this because if we didn't, the user would keep sending the POST request upon every refresh or page change.

However; if we do it right now, we will lose context, and we won't know if the Action was 'Add-Approver' or 'Delete' or simply we won't know if it was a POST request since for every POST request we force a GET request using 

`$th.is->getOutput()->redirect( $this->getPageTitle()->getLocalURL() );`

A small JS script is taking care of resending POST requests upon page refresh.